### PR TITLE
Prepare for 12.3.0 release

### PR DESCRIPTION
**Pull request by PeterSuna**
Release notes:
Update upstream dependencies to latest Magnesium SR3 versions:

- odl parent 6.0.13
- yangtools-artifacts 4.0.15
- mdsal 5.0.16
- controller 1.11.3
- conroller.config-artifacts 0.12.3
- aaa 0.11.3
- netconf 1.8.3
- infrautils  1.7.3
- openflow 0.10.3
- serviceutils 0.5.3
- maven-sal-api-gen-plugin 3.0.16

### CODE REVIEW

**Review by vainyksi — CHANGES_REQUESTED** [review]
please introduce `12.3.0-SNAPSHOT` instead of commit https://github.com/PANTHEONtech/lighty/pull/642/commits/6a9dbd57d190c20f35b908076b525e1963694782
those version change to `12.3.0` will be held by release plugin

**Review by vainyksi — APPROVED** [review]

File: lighty-core/dependency-versions/pom.xml [hunk 1]
```diff
@@ -34,63 +34,63 @@
...
-                <version>0.5.2</version>
+                <version>0.5.3</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>org.opendaylight.yangtools</groupId>
                 <artifactId>yangtools-artifacts</artifactId>
-                <version>4.0.13</version>
+                <version>4.0.14</version>
```
Comment (rovarga) [inline lighty-core/dependency-versions/pom.xml:93]: pick 4.0.15 for: https://jira.opendaylight.org/secure/ReleaseNote.jspa?projectId=10188&version=11716

File: lighty-core/dependency-versions/pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
             <dependency>
                 <groupId>org.opendaylight.odlparent</groupId>
                 <artifactId>odlparent</artifactId>
-                <version>6.0.11</version>
+                <version>6.0.12</version>
```
Comment (rovarga) [inline lighty-core/dependency-versions/pom.xml:28]: pick 6.0.13 for https://github.com/opendaylight/odlparent/blob/6.0.x/docs/NEWS.rst#version-6013

**Review by rovarga — CHANGES_REQUESTED** [review]

File: lighty-core/dependency-versions/pom.xml (hunk 1 above)
Comment (rovarga) [reply lighty-core/dependency-versions/pom.xml:93]: not important, but keeps things neater

Comment (PeterSuna) [reply lighty-core/dependency-versions/pom.xml]: Done, also I have updated yang-maven-plugin to 4.0.15

File: lighty-core/dependency-versions/pom.xml (hunk 2 above)
Comment (PeterSuna) [reply lighty-core/dependency-versions/pom.xml]: Done, also I have updated checkstyle to 6.0.13

**Review by rovarga — APPROVED** [review]

**Pull request merged**